rllib/algorithms/algorithm_config.py

```diff
@@ -769,14 +774,6 @@ def validate(self) -> None:
                 "`config.evaluation(enable_async_evaluation=True) on your config "
                 "object to fix this problem."
             )
-        if not (
```

Comment:
Moved this below into `rollouts()` method to check more immediately (this is only a single arg check after all).